src/main/scala/com/imaginea/activegrid/core/models/AnsiblePlayBookRunner.scala

```diff
@@ -1,24 +1,42 @@
 package com.imaginea.activegrid.core.models
+
 import com.typesafe.scalalogging.Logger
 import org.slf4j.LoggerFactory
-import scala.util.Try
+
+import scala.util.{Failure, Success}
+
 /**
   * Created by sivag on 17/1/17.
   */
 class AnsiblePlayBookRunner(workflowContext: WorkflowContext) {
   val logger = Logger(LoggerFactory.getLogger(AnsiblePlayBookRunner.this.getClass.getName))
-  def executePlayBook() : Try[Boolean] = {
-    //todo implementation.
-    Try(false) //dummy response.
-  }
-  def execute() : Try[Boolean] = {
-     executePlayBook() recover {
-       case e : Exception =>
-         val workflowId = WorkflowContext.get().workflow.id
-       logger.error(s"An unexpected error, Execution of workflow ($workflowId) aborted., Removig workflow from running list",e)
-       CurrentRunningWorkflows.remove(workflowId.getOrElse(0L));
-     }
-    Try(true);
+
+  def execute(): Unit = {
+    try {
+      val workflow = workflowContext.workflow
+      val execution = workflow.execution.get
```

Comment:
Do we know it can't be None? If we do why is it optional? 
If it's some local knowledge, add a comment.